test/controllers/api/docs_controller_test.exs

```diff
@@ -8,6 +8,8 @@ defmodule HexWeb.API.DocsControllerTest do
   setup do
     User.create(%{username: "eric", email: "eric@mail.com", password: "eric"}, true)
     |> HexWeb.Repo.insert!
+    File.rm_rf!(Application.fetch_env!(:hex_web, :tmp_dir))
+    File.mkdir!(Application.fetch_env!(:hex_web, :tmp_dir))
```

Comment:
This can break async tests so another solution is needed. Not really sure exactly what is breaking for this, can't you delete the specific conflicting file(s)?